Makefile

```diff
@@ -10,3 +10,32 @@ all:
 .PHONY: clean
 clean:
 	@-go clean -i
+
+.PHONY: ci
+ci: check-gofmt check-goimports check-golint vet test test-race
+
+# goimports doesn't support the -s flag to simplify code, therefore we use both
+# goimports and gofmt -s.
+.PHONY: check-gofmt
+check-gofmt:
+	bash -c 'diff <(gofmt -s -d .) <(printf "")'
```

Comment:
@david-martin check this out, just learned a nicer way to make sure `gofmt` etc fail if the output is not empty :)